tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
+    "importHelpers": true,
```

Comment:
`importHelpers` requires TypeScript [v2.1](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-1.html#support-for-external-helpers-library-tslib)